app/server.go

```diff
@@ -213,10 +213,10 @@ func NewServer(options ...Option) (*Server, error) {
 		}
 	})
 
-	mlog.Info(fmt.Sprintf("Current version is %v (%v/%v/%v/%v)", model.CurrentVersion, model.BuildNumber, model.BuildDate, model.BuildHash, model.BuildHashEnterprise))
-	mlog.Info(fmt.Sprintf("Enterprise Enabled: %v", model.BuildEnterpriseReady))
+	mlog.Info("Current ", mlog.String("version", model.CurrentVersion), mlog.String("buildNumber", model.BuildNumber), mlog.String("buildDate", model.BuildDate), mlog.String("buildHash", model.BuildHash), mlog.String("enterprise", model.BuildHashEnterprise))
+	mlog.Info("Enterprise ", mlog.String("Enabled", model.BuildEnterpriseReady))
 	pwd, _ := os.Getwd()
-	mlog.Info(fmt.Sprintf("Current working directory is %v", pwd))
+	mlog.Info("Current working  ", mlog.String("directory", pwd))
```

Comment:
I don't think this change is needed, it doesn't improve logging

Reply:
@reflog According to the issue the PR is referencing, it suggested to replace the `fmt.Sprintf` used in logging, is there something I might have missed?

Reply:
This one is a bit odd to me because the log message is just `Current working` which could be confusing. I could see leaving this as is, or doing something like this:
```suggestion
	mlog.Info("Printing current working directory", mlog.String("directory", pwd))
```

Reply:
@gabrieljackson Thanks for this, I would work on it.